spec/stat_tracker_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'simplecov'
```

Comment:
All we should need here is `require 'spec_helper'`, I think it could possibly cause some problems to initialize SimpleCov in more than one location.